### DISCUSSION TITLE
I would like to use the professional version.

### DISCUSSION THREAD

**Discussion by jarvischen666** [Codium-ai/cover-agent#225] [General]
I would like to use the professional version. We previously inquired about collaboration matters regarding your product, Cover-Agent, but we have not received a response from the QODO platform. Could you please let us know if there are any other ways to contact this platform besides the "CONTACT US" option on the website? Thank you.

**Comment by DedyKredo** [comment]
@jarvischen666 Sorry you didn't get a response! I'll make sure we get back to you ASAP
